integration_tests/mockApis/auth.ts

```diff
@@ -156,7 +156,14 @@ const stubUserRoles = () =>
 export default {
   getSignInUrl,
   stubAuthPing: ping,
-  stubSignIn: (): Promise<[Response, Response, Response, Response, Response, Response]> =>
-    Promise.all([favicon(), redirect(), signOut(), manageDetails(), token(), tokenVerification.stubVerifyToken()]),
+  stubSignIn: (authorities: string[]): Promise<[Response, Response, Response, Response, Response, Response]> =>
```

Comment:
The cypress task `stubSignIn` is a logged in user, but with no roles at all (see the empty collection of `authorities` as the function arg)

I decided against renaming the stub to something like `stubSigninAsUserWithNoAuthorities` because `stubSignIn` comes from the HMPPS template, and all UI apps will use it. So for consistency I decided to leave the name as it was on the basis that it would be more confusing for future devs to not find a stub called `stubSignIn` at all